### ISSUE TITLE
request for grid plot support for sweep

### ISSUE THREAD

**Issue by kengz** [wandb/client#1055]
`wandb --version && python --version && uname`

* Weights and Biases version:
* Python version:
* Operating System:

### Description

The parallel plot for sweep is useful for visualizing hyperparameters but it can be hard to choose the best hyperparameters from it, because the plots are sometimes too crowded so it can be hard to tell, and there's no natural ordering by the best param value. A more natural complementary plot to it is the grid plot - simply a grid of subplots with y-axis as the metric ("val_loss"), and multiple plots with x-axis, one for each hyperparameter.

For example below, we have multiple y-axis to evaluate the best parameters judged by stability, strength(like accuracy), etc. The color gradient indicates the primary metric (fitness in this case). With this graph we can clearly tell which values tend to be better on average, more stable, more spread out, etc. Since WandB streams and updates the datapoints, we should be able to see the points move towards better score as the sweep algorithm chooses better hyperparams.

![ddqn_epsilon_poly_lunar_experiment_graph](https://user-images.githubusercontent.com/5512945/49716803-b8322780-fc09-11e8-9a99-c4287ddc28e0.png)

>Image taken from source: https://github.com/kengz/SLM-Lab/pull/251

I think this could be useful also for other purposes. Please consider supporting it.

**Comment by cvphelps** [comment]
Thanks for this feature request, we have more complex plot support in our roadmap.

**Comment by ariG23498** [comment]
Hey @kengz 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
